AppDB/cassandra_env/cassandra_interface.py

```diff
@@ -165,16 +171,23 @@ def batch_delete(self, table_name, row_keys, column_names=[]):
     if not isinstance(table_name, str): raise TypeError("Expected a str")
     if not isinstance(row_keys, list): raise TypeError("Expected a list")
 
-    path = ColumnPath(table_name)
+    row_keys_bytes = [bytearray(row_key) for row_key in row_keys]
+
+    statement = 'DELETE FROM "{table}" WHERE {key} IN %s'.\
+      format(
+        table=table_name,
+        key=ThriftColumn.KEY
+      )
+    query = SimpleStatement(statement,
+                            consistency_level=ConsistencyLevel.QUORUM)
```

Comment:
I believe it would be better to set the consistency level as a constant so we can easily swap QUORUM out for ONE if need be.

Does that make sense?